src/actions/mover.ts

```diff
@@ -123,6 +124,7 @@ export class Mover {
     // Begin drag.
     dragger.onDragStart(info.fakePointerEvent('pointerdown'));
     info.updateTotalDelta();
+    block.addIcon(new MoveIcon(block));
```

Comment:
What's the reason to add the icon here instead of in the keyboard drag strategy?

Reply:
It was the first place I ran across to do it, but agreed that makes more sense; moved it accordingly.